src/components/upgrader/boxfishUpgraderFactory.ts

```diff
@@ -18,7 +18,11 @@ export async function createBoxfishUpgrader(
   logger: any
 ): Promise<IDeviceUpgrader> {
   const info = await manager.getInfo();
-  if (semver.gte(info.version, HPK_SUPPORT_VERSION)) {
+  let HPK_SUPPORT_VERSION_VAR = HPK_SUPPORT_VERSION;
```

Comment:
You could also do  export const HPK_SUPPORT_VERSION =   process.env.HPK_SUPPORT_VERSION || '1.2.1-0';